src/atoms/Checkbox/Checkbox.native.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { TouchableOpacity } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import CheckBoxIcon from './CheckboxIcon';
+import Text from '../Text';
+import Flex from '../Flex';
+import { isEmpty } from '../../_helpers/utils';
+
+const styles = {
+  backdrop: {
+    width({ width }) {
+      return width;
+    },
+    height({ height }) {
+      return height;
+    },
+    borderRadius({ borderRadius }) {
+      return borderRadius;
+    },
+    backgroundColor({ backgroundColor }) {
+      return backgroundColor ? backgroundColor : 'transparent';
+    },
+  },
+};
+
+const Backdrop = styled.View`
+  width: ${styles.backdrop.width}px;
+  height: ${styles.backdrop.height}px;
+  border-radius: ${styles.backdrop.borderRadius}px;
+  background-color: ${styles.backdrop.backgroundColor};
+`;
+
+const TextContainer = styled.View`
+  margin-left: 4px;
+`;
+
+const mapSizeToBackdropProps = (checkBoxSize) => {
+  switch (checkBoxSize) {
+    case 'large':
+      return 24;
+    case 'medium':
+      return 20;
+    case 'small':
+      return 16;
+    case 'xsmall':
+      return 12;
+    default:
+      return 20;
+  }
+};
+
+const mapSizeToHelpTextSizeProp = (checkboxSize) => {
+  switch (checkboxSize) {
+    case 'large':
+      return 'medium';
+    case 'medium':
+      return 'xsmall';
+    default:
+      return 'small';
+  }
+};
+
+const Checkbox = ({ onChange, checked, disabled, size, title, helpText, testID }) => {
+  const [underlayColor, setUnderlayColor] = useState('');
```

Comment:
Should we have a default underlayColor? Empty string is not a valid color

Reply:
Added "transparent" as default underlay color.